SB2018-volunteer/2018-04-13.md

```diff
@@ -0,0 +1,40 @@
+---
+to: All Volunteers
+subject: Volunteer Information - SourceBots 2018
+---
+
+First of all, on behalf of everyone at SourceBots we would like to say a **massive** thank you for volunteering at the competition! This event could not run without you and we appreciate the hardwork and effort that you will put in!
+
+Even though it is hard work, we can assure you that it will be a fun weekend filled with robot madness and making sure everything is running smoothly!
+
+To that end, we thought now would be a good time to send an email detailing information about the competition! :)
```

Comment:
Unnecessary smiley

Reply:
fixed in 9d736b9d45515412a9b0c036d0959422dfef7b3b